torch/csrc/jit/mobile/interpreter.h

```diff
@@ -14,6 +14,7 @@ struct Code {
   std::vector<std::function<void(Stack&)>> operators_;
   std::vector<c10::IValue> constants_;
   std::vector<c10::TypePtr> types_;
+  std::vector<std::shared_ptr<torch::jit::Function>> custom_class_fns_;
```

Comment:
Not sure if it couples with the full JIT. In internal BUCK build the JIT codes are carefully decoupled for lite interpreter. Could you import to internal and try to build //xplat/caffe2:torch_mobile_core?

Reply:
This PR decoupled Function from the JIT https://github.com/pytorch/pytorch/pull/33921